pro/cypress/e2e/step-definitions/signupJourneyCreateOfferer.cy.ts

```diff
@@ -88,13 +49,21 @@ When('I specify an offerer with a SIRET', () => {
     method: 'GET',
     url: `/venues/siret/${mySiret}`,
   }).as('venuesSiret')
+  cy.intercept(
+    'GET',
+    'https://api-adresse.data.gouv.fr/search/?limit=1&q=*'
+  ).as('search1Address')
   cy.findByText('Continuer').click()
-  cy.wait(['@getSiret', '@venuesSiret']).then((interception) => {
-    if (interception[0].response)
-      expect(interception[0].response.statusCode).to.equal(200)
-    if (interception[1].response)
-      expect(interception[1].response.statusCode).to.equal(200)
-  })
+  cy.wait(['@getSiret', '@venuesSiret', '@search1Address']).then(
+    (interception) => {
+      if (interception[0].response)
+        expect(interception[0].response.statusCode).to.equal(200)
+      if (interception[1].response)
+        expect(interception[1].response.statusCode).to.equal(200)
+      if (interception[2].response)
+        expect(interception[2].response.statusCode).to.equal(200)
```

Comment:
il y a un ptit problème de linter
on prefère écrire 
```suggestion
      if (interception[0].response) {
        expect(interception[0].response.statusCode).to.equal(200)
     }
```

Reply:
où vois-tu un problème de linter ? Tout est vert dans l'onglet checks...

Reply:
@scolson-pass Est-ce que le prettier de ton editeur de code a bien le bon path vers la config ".prettierrc" ? Parce que tu devrais voir les erreurs dans le code directement!

Sur VSCode c'est dans Settings -> Prettier config path, et moi j'ai pour valeur "pro/.prettierrc"

Reply:
OK c'est corrigé. Merci pour le retour